projects/demo-integrations/cypress/tests/demo/exclusions.ts

```diff
@@ -0,0 +1,10 @@
+const map = new Map([
+    ['components/select', [5, 6]],
```

Comment:
Why are we excluding those?

Reply:
I remove the 6th example from the exclusion. 5th example makes the tests unstable due to the loader.